ui/src/dataLoaders/components/verifyStep/TelegrafInstructions.tsx

```diff
@@ -55,6 +56,10 @@ class TelegrafInstructions extends PureComponent<Props> {
       </div>
     )
   }
+
+  private get origin(): string {
+    return window.location.origin
```

Comment:
I don't think will work in cloud, we need to specify an exact url.

Reply:
@goller says this should work just fine for now in cloud